clients/http/utils/common.go

```diff
@@ -146,6 +146,20 @@ func createRequestWithRawData(ctx context.Context, httpMethod string, baseUrl st
 	return req, nil
 }
 
+func CreateRequestWithRawDataAndHeaders(ctx context.Context, httpMethod string, baseUrl string, requestPath string, requestParams url.Values, data any, headers map[string]string) (*http.Request, errors.EdgeX) {
```

Comment:
this one can keep the lower case

Reply:
I just exported this func so that we reuse it create request, such as
https://github.com/IOTechSystems/go-mod-central-ext/pull/7/commits/0012cb66afbe6e2ba41a5e9879f9378ab577a397#diff-35b9402dffb917185edf0d25a06f117f13be6554f193ec7e71e0fb2be7d785e9R36

Reply:
ok, lets turn all other createRequestXXX to upper cases

Reply:
Done